P/PETSc/build_tarballs.jl

```diff
@@ -11,6 +11,11 @@ SUITESPARSE_COMPAT_VERSION = "5.10.1"
 SUPERLUDIST_COMPAT_VERSION = "8.1.2"   
 MPItrampoline_compat_version="5.2.1"    
 
+SCALAPACK32_COMPAT_VERSION="2.2.1"
```

Comment:
These compat declarations assume that all these pacakges use semver, and that PETSc can be built with all other versions that are compatible according to semver. Is that true?

Reply:
In this particular case it no longer compiles if I don’t add this as superlu_dist is only compatible with a certain version of scotch, which was updated meanwhile. 
Drives me nuts, but I guess it is the only way to make it work.

Reply:
Does this mean you need to add an `=` sign in front of the version number?

Reply:
yes I think so (unless I don't understand you comment)